sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -1,3 +1,189 @@
-# Azure Messaging EventHub service
+<!-- cspell:words godoc azeventhubs  -->
+# Azure Event Hubs Client Package for C++
 
-C++ SDK for Azure Messaging EventHub.
+[Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) is a big data streaming platform and event ingestion service from Microsoft. For more information about Event Hubs see: [link](https://docs.microsoft.com/azure/event-hubs/event-hubs-about).
+
+Use the client library `github.com/Azure/azure-sdk-for-cpp/sdk//eventhubs` in your application to:
+
+- Send events to an event hub.
+- Consume events from an event hub.
+
+Key links:
+- [Source code][source]
+- [API Reference Documentation][godoc]
```

Comment:
GoDoc? Let's update/remove.

Reply:
As I mentioned in our team meeting, the links to go are intentional. They will be fixed after we ship the package and the documentation is published.